var/spack/repos/builtin/packages/fujitsu-mpi/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class FujitsuMpi(Package):
+    """Fujitsu MPI implementation only for Fujitsu compiler."""
+
+    homepage = "https://www.fujitsu.com/us/"
+
+    version('3.0')
```

Comment:
Spack can't guess a fetch method for this version since there is no URL or git. I would just remove the version number. If it's only going to be used as an external package, the user will have to define the version number in their `packages.yaml` anyway.

Reply:
Thanks grateful!! I understood that versions could be defined in `package.yaml` when no definition of versions in `package.py`. I deleted definition of version in `package.py` on latest commit.